### PR TITLE
Improve the test coverage

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related Issue: [#1288](https://github.com/ballerina-platform/ballerina-standard-library/issues/1288)

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: random-ballerina/random_errors.bal [hunk 1]
```diff
@@ -16,3 +16,6 @@
 
 # Represents Random module related errors.
 public type Error distinct error;
+
+# Represents the arithmetic error.
+public type ArithmeticError distinct Error;
```
Comment (MadhukaHarith92) [inline random-ballerina/random_errors.bal:21]: Do we use this anywhere

Comment (kalaiyarasiganeshalingam) [reply random-ballerina/random_errors.bal:21]: Yes, We are creating an arithmetic error on the native side but Ballerina has only a distinct error as `Error`. So Ballerina can't catch that error.

- https://github.com/ballerina-platform/module-ballerina-random/blob/main/random-native/src/main/java/org/ballerinalang/stdlib/random/nativeimpl/ExternMethods.java#L49
- https://github.com/ballerina-platform/module-ballerina-random/blob/main/random-native/src/main/java/org/ballerinalang/stdlib/random/nativeimpl/Constant.java#L28

Comment (MadhukaHarith92) [reply random-ballerina/random_errors.bal:21]: Ack.

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Pull request merged**